packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -0,0 +1,69 @@
+# Kontrol Verification
+
+This folder contains Kontrol symbolic property tests.
+
+## Directory structure
+
+The directory structure is as follows
+
+```tree
+test/kontrol
+├── Counter.sol
+├── Counter.t.sol
+├── kontrol
```

Comment:
Nit, can we give a different name to this folder? `test/kontrol/kontrol` feels confusing until you read the readme. I was thinking `utils` is a good name, but we also have a `test/kontrol/proofs/utils` so maybe that's not a great choice either 🤔

Reply:
Hm makes sense. What about `project`, `general`, or `kontrol-utils`?

Reply:
Perhaps this second `kontrol` folder should just be removed and its files can live directly in `test/kontrol`? It seems pretty generic so instead having `test/kontrol/scripts` and `test/kontrol/pausability-lemmas.k` feels clean

Reply:
Agreed!

Reply:
I still need to update this part of the README